common/fs/src/cache/watch.rs

```diff
@@ -1,417 +0,0 @@
-use inotify::{EventMask, Inotify, WatchDescriptor, WatchMask};
```

Comment:
Is this file just leftover dead code?

Reply:
yes, left over after porting from inode to cross platform notify-rs.